doc/languages-frameworks/python.section.md

```diff
@@ -144,8 +144,10 @@ The `buildPythonPackage` mainly does four things:
   environment variable and add dependent libraries to script's `sys.path`.
 * In the [`installCheck`](#ssec-installCheck-phase) phase, `${python.interpreter} -m pytest` is run.
 
-By default tests are run because [`doCheck = true`](#var-stdenv-doCheck). Test dependencies, like
-e.g. the test runner, should be added to [`nativeCheckInputs`](#var-stdenv-nativeCheckInputs).
+By default tests are run because [`doInstallCheck = true`](#var-stdenv-doInstallCheck). Test dependencies, like
+e.g. the test runner, should be added to [`nativeInstallCheckInputs`](#var-stdenv-nativeInstallCheckInputs).
+
+Python packages and applicationst doesn't have [`checkPhase`](#ssec-check-phase). `doCheck`, `checkPhase`, `checkInputs`, and `nativeCheckInputs` are aliases to the `installCheck`-related attributes for historical reason, but using them instead of the `installCheck`-related variables are obsolete and discouraged.
```

Comment:
Minor spelling and grammar suggestion:
```suggestion
Python packages and applications don't have [`checkPhase`](#ssec-check-phase). `doCheck`, `checkPhase`, `checkInputs`, and `nativeCheckInputs` are aliases to the `installCheck`-related attributes for historical reason, but using them instead of the `installCheck`-related variables are obsolete and discouraged.
```